setup.py

```diff
@@ -1,9 +1,11 @@
 from distutils.core import setup
 
+from setuptools import find_packages
+
 setup(
     name='django-auditlog',
     version='0.4.5',
-    packages=['auditlog', 'auditlog.migrations', 'auditlog.management', 'auditlog.management.commands'],
+    packages=find_packages(where='src', exclude=['*tests*', 'runtests']),
```

Comment:
I am ok switching to `find_packages` but I am concerned that the `exclude` is only excluding `src/auditlog_tests/tests.py` and not the entire `src/auditlog_tests`.

Reply:
I can tweak it.